test-buildpacks/readonly-layer-files/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "readonly-layer-files"
 version = "0.0.0"
-edition = "2021"
-rust-version = "1.60"
+edition.workspace = true
+rust-version.workspace = true
 publish = false
 
 [dependencies]
```

Comment:
This and the other test-buildpack depend on libcnb via explicit dependencies, but could be switched to using inherited dependencies too.

(The same applies to the example buildpacks, but for the examples I think we want them to be more easily copy-pasteable by end users; whereas these test-buildpacks presumably aren't intended for use as examples?)

Reply:
I mean, the examples aren't very copy & paste friendly anyway with the repository-local dependency. Since users wanting to c&p an example, with both the current solution and the workspace one, need to replace everything coming after `libcnb` in the `Cargo.toml`, I lean towards using the workspace dependency there as well.

Reply:
Yeah agree :-)